datahub-web-react/src/graphql/schemaBlame.graphql

```diff
@@ -0,0 +1,46 @@
+query getSchemaBlame($input: GetSchemaBlameInput!) {
+    getSchemaBlame(input: $input) {
+        latestVersion {
+            semanticVersion
+            semanticVersionTimestamp
+            versionStamp
+        }
+        version {
+            semanticVersion
+            semanticVersionTimestamp
+            versionStamp
+        }
+        semanticVersionList {
+            semanticVersion
+            semanticVersionTimestamp
+            versionStamp
+        }
+        schemaFieldBlameList {
+            fieldPath
+            schemaFieldChange {
+                timestampMillis
+                actor
+                semanticVersion
+                semanticVersionChange
+                changeType
+                lastSchemaFieldChange
+                versionStamp
```

Comment:
What is this thing used for ? If we don't explicitly need it we should remove (not sure if we do)

Reply:
Removing fields we don't need, good call